azext_edge/edge/_help.py

```diff
@@ -443,6 +443,22 @@ def load_iotops_help():
 
                       After the foundation layer has been installed the `az iot ops create` command should
                       be used to deploy an instance.
+
+                      Note: --*-config options allow override of default config settings.
+
+                      The default config settings for container storage are:
+                        edgeStorageConfiguration.create=true
+                        feature.diskStorageClass=default,local-path
+
+                      if --enable-fault-tolerance is used the following delta applies:
```

Comment:
If

also how would this interact with custom config settings (prob note here too)

Reply:
Talked offline.